src/olympia/accounts/tests/test_serializers.py

```diff
@@ -19,10 +19,16 @@ def setUp(self):
     def serialize(self):
         return self.serializer(self.user).data
 
-    def test_picture_url(self):
+    def test_picture(self):
         serial = self.serialize()
         assert ('anon_user.png' in serial['picture_url'])
         assert serial['picture_type'] == ''
+        assert 'picture_upload' not in serial  # its a write only field.
+
+        self.user.update(picture_type='image/jpeg')
+        serial = self.serialize()
+        assert serial['picture_url'] == self.user.picture_url
+        assert '%s.png' % self.user.id in serial['picture_url']
```

Comment:
I'd like to actually check it was written but is that possible in tests?  (I assumed either the resize task wouldn't fire or writes don't happen in tests)